src/camera.c

```diff
@@ -853,9 +880,133 @@ void func_80068508(s32 arg0) {
 }
 
 GLOBAL_ASM("asm/non_matchings/camera/func_80068514.s")
-GLOBAL_ASM("asm/non_matchings/camera/func_80068BF4.s")
+
+typedef struct unk80068BF4 {
```

Comment:
Should the struct definition and GFX macros be moved elsewhere?

Reply:
I shall fix that

Reply:
I have fixed that.